testdata/jsight_0.3/ser-to-openapi/req.japi.type_0.3/user_type_negatives.jst

```diff
@@ -1,6 +1,6 @@
 JSIGHT 0.3
 
-TYPE @ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789___1_-_-_1 // Long description with different symbols... tabs:			ascii: !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~ spec:¹²³$‰↑∞←→—≠€®™ѣѵіѳ′[]≈§°£₽„“”‘’×©↓−«»…´“„•ẞˇ¢·¸¨‘⌀⌘˘˚ѢѴІѲ″{}±–〉〈¿ˆ¼⅓½¡ Cyrillic: йцукенгшщзФЫВАПРОЛДЖЭ Chinese: 你好世界！
```

Comment:
Why did you  change the test? Just leave it as it is and put the whole file as "*.todo"

Reply:
fixed, but '#' inside single line annotation will stop tests after uncomment todo